language-support/ts/daml-ledger/index.test.ts

```diff
@@ -0,0 +1,243 @@
+// Copyright (c) 2020 The DAML Authors. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import { Template, ContractId, Choice } from "@daml/types";
+import WS from "jest-websocket-mock";
+import Ledger from "./index";
+import { Event, CreateEvent } from "./index";
+import * as jtv from "@mojotech/json-type-validation";
+
+type Foo = {};
+
+const Foo: Template<Foo, undefined, "foo-id"> = {
+  templateId: "foo-id",
+  keyDecoder: () => jtv.constant(undefined),
+  decoder: () => jtv.object({}),
+  Archive: ({} as unknown) as Choice<Foo, {}, {}, undefined>,
```

Comment:
The TS compiler is happy without the parentheses:
```suggestion
  Archive: {} as unknown as Choice<Foo, {}, {}, undefined>,
```